config/locales/es.yml

```diff
@@ -2048,6 +2048,82 @@ es:
           verification_code_label: 'Introduzca el código de 6 números '
           verify: Verifique
     ctc_pages:
+      07_info_required_by_irs: Recopilamos información según lo dispuesto por el IRS en el Procedimiento de Ingresos "Rev RP-21-24".
```

Comment:
The `07_info_required_by_irs` key here seems to be nested directly under `ctc_pages`. In the English, it's nested under `privacy_policy`. I think the Spanish keys here need to move.